instrumentation/net/http/otelhttp/internal/semconvutil/httpconv.go

```diff
@@ -291,18 +301,35 @@ func (c *httpConv) ClientRequest(req *http.Request) []attribute.KeyValue {
 // The req Host will be used to determine the server instead.
 //
 // The following attributes are always returned: "http.method", "http.scheme",
-// "http.flavor", "http.target", "net.host.name". The following attributes are
-// returned if they related values are defined in req: "net.host.port",
-// "net.sock.peer.addr", "net.sock.peer.port", "http.user_agent", "enduser.id",
-// "http.client_ip".
+// "http.target", "net.host.name". The following attributes are returned if they
+// related values are defined in req: "net.host.port", "net.sock.peer.addr",
+// "net.sock.peer.port", "user_agent.original", "http.client_ip",
+// "net.protocol.name", "net.protocol.version".
 func (c *httpConv) ServerRequest(server string, req *http.Request) []attribute.KeyValue {
-	// TODO: This currently does not add the specification required
-	// `http.target` attribute. It has too high of a cardinality to safely be
-	// added. An alternate should be added, or this comment removed, when it is
-	// addressed by the specification. If it is ultimately decided to continue
-	// not including the attribute, the HTTPTargetKey field of the httpConv
-	// should be removed as well.
-
+	/* The following semantic conventions are returned if present:
+	http.method				string
+	http.scheme				string
+	net.host.name			string
+	net.host.port			int
+	net.sock.peer.addr		string
+	net.sock.peer.port		int
+	user_agent.original		string
+	http.client_ip			string
+	net.protocol.name		string
```

Comment:
This should rather be removed. See

> Recommended: if not default (http)

Reply:
This can be something else.  The default server will always be HTTP, but it can respond to spdy or other protocols.  So it does set it if it's not the default.